models/spring-ai-ollama/src/main/java/org/springframework/ai/ollama/api/OllamaModel.java

```diff
@@ -51,6 +52,16 @@ public enum OllamaModel implements ChatModelDescription {
 	 */
 	QWEN3_4B("qwen3:4b"),
 
+	/**
+	 * Qwen3 1.7b
+	 */
+	QWEN_3_1_7_B("qwen3:1.7b"),
+
+	/**
+	 * Qwen3 0.6b
+	 */
+	QWEN_3_06B("qwen3:0.6b"),
+
```

Comment:
Among the existing constants, there is QWEN3_4B, while the newly added ones are QWEN_3_1_7_B and QWEN_3_06B. The naming styles are inconsistent (such as the use of underscores and the separation of numbers and units). It is recommended to adopt a unified style (for example, using QWEN3_1_7B / QWEN3_0_6B consistently or applying a uniform underscore scheme) to ensure readability and searchability.